security-analysis/security-analysis-default/src/main/java/com/powsybl/security/impl/DefaultSecurityAnalysis.java

```diff
@@ -296,16 +297,18 @@ private void checkPreContingencyViolations(Network network, Consumer<LimitViolat
         if (violationDetector != null) {
             violationDetector.checkAll(network, consumer);
         } else {
-            LimitViolationDetection.checkAll(network, EnumSet.allOf(LoadingLimitType.class), 1., consumer);
+            LimitViolationDetection.checkAll(network, EnumSet.allOf(LoadingLimitType.class), new LimitsComputer.NoModificationsImpl(), consumer);
         }
     }
 
     protected void checkPostContingencyViolations(Contingency contingency, Network network, Consumer<LimitViolation> consumer) {
         if (violationDetector != null) {
             violationDetector.checkAll(contingency, network, consumer);
         } else {
-            // TODO: Take the contingency into account with full support of LimitReductions.
-            LimitViolationDetection.checkAll(network, EnumSet.allOf(LoadingLimitType.class), 1., consumer);
+            // TODO For now, contingencies are ignored in the default security analysis
+            //  (same behavior as using the default LimitViolationDetector)
+            //  This should change to fully support LimitReductions
```

Comment:
Use the comment present in `main` (I think you made a mistake while rebasing your PR):
```suggestion
            // TODO: Take the contingency into account with full support of LimitReductions.
```